--- a/vendor/magento/module-sales/Controller/Adminhtml/Order/AddComment.php
+++ b/vendor/magento/module-sales/Controller/Adminhtml/Order/AddComment.php
@@ -55,7 +55,7 @@
                 }
 
                 $comment = trim(strip_tags($data['comment']));
-                $history = $order->addStatusHistoryComment($comment, $orderStatus);
+                $history = $order->addStatusHistoryComment($comment, $data['status']);
                 $history->setIsVisibleOnFront($visible);
                 $history->setIsCustomerNotified($notify);
                 $history->save();
